README.md

```diff
@@ -31,7 +31,54 @@ Trestle provides tooling to help orchestrate the compliance process across a num
 
 ## Important Note:
 
-The current version of trestle supports NIST OSCAL 1.1.2 as well as previous versions 1.1.x and 1.0.x. All files created by trestle will be output as OSCAL version 1.1.2.
+The current trestle v3:
+
+- supports NIST OSCAL 1.1.2 as well as previous versions
+- new OSCAL json documents created will be output as OSCAL version 1.1.2
+- old OSCAL json documents created using trestle v2 are supported (backwards compatibility)
+- some Python module dependencies have been upgraded
+
+OSCAL changes between 1.0.4 and 1.1.2 can be found here: [https://github.com/usnistgov/OSCAL/releases](https://github.com/usnistgov/OSCAL/releases). In short:
```

Comment:
Wondering if migration information a better addition to the `trestle v3` announcement discussion or maybe the release notes for the major version. What do you think?

Reply:
The details should be stricken? The link to OSCAL changes info is OK? Or something else?

Reply:
Thanks for the question here @degenaro.

I thought about this more and would like to clarify my suggestion and make it more concrete. The motivation behind it is to keep the `README.md` concise and avoid information overload for new users. If this is a format you agree with, I am also happy to help author this as a follow on PR instead.

`README.md`
```
## Development status

### v3: stable

Version `3` is actively developed and is based on NIST OSCAL version 1.1.2.

Features can be found below:
- supports NIST OSCAL 1.1.2 as well as previous versions
- new OSCAL json documents created will be output as OSCAL version 1.1.2
- old OSCAL json documents created using trestle v2 are supported (backwards compatibility)

Details on changes from v2 to v3 can be found in our [MIGRATION_GUIDE.md](./MIGRATION_GUIDE.md)

### v2: stable (maintenance mode) - or leave this out if desired
```

`MIGRATION_GUIDE.md` - version specific details
```
v2 to v3 details and OSCAL migration details here...
```

Reply:
Your points are well taken. I have revised accordingly, partially employing your suggestion.

After further consideration, I've removed the `Important Note` due to old version of trestle no longer supported.